docs/release-info/release-notes.md

```diff
@@ -461,12 +536,23 @@ Parallel compaction task completion reports now have `segmentsRead` and `segment
 
 [#15947](https://github.com/apache/druid/pull/15947)
 
+#### Changes to segment schema cleanup default values
+
+The following are the changes to the default values of segment schema cleanup:
+
+* The default value for `druid.coordinator.kill.segmentSchema.period` has changes from `PT1H` to `P1D`.
+* The default value for `druid.coordinator.kill.segmentSchema.durationToRetain` has changed from `PR6H` to `P90D`.
+
+[#16354](https://github.com/apache/druid/pull/16354)
+
```

Comment:
This section is not needed since these configs are being newly added in Druid 30 itself.

Reply:
I'll add it to the Centralized datasource schema (alpha) section.